adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/CoverageRDD.scala

```diff
@@ -124,19 +133,35 @@ case class ParquetUnboundCoverageRDD private[rdd] (
     val sqlContext = SQLContext.getOrCreate(sc)
     import sqlContext.implicits._
     sqlContext.read.parquet(parquetFilename)
-      .select("contigName", "start", "end", "score")
+      .select("contigName", "start", "end", "score", "sampleId")
       .withColumnRenamed("score", "count")
       .as[Coverage]
   }
 
   def toFeatures(): FeatureRDD = {
-    ParquetUnboundFeatureRDD(sc, parquetFilename, sequences)
+    ParquetUnboundFeatureRDD(sc, parquetFilename, sequences, samples)
   }
 
+  /**
+   * Replaces the sequence dictionary attached to a CoverageRDD.
+   *
+   * @param newSequences The new sequence dictionary to attach.
+   * @return Returns a new CoverageRDD with the sequences replaced.
+   */
   def replaceSequences(
```

Comment:
This method is defined with scaladoc in `GenomicDataset`, could we use `override def` instead?